# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdissert/compile-fix.patch
# Copyright (C) 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur kdissert-1.0.5-orig/src/kdissert/canvasview/DCanvasLink.cpp kdissert-1.0.5/src/kdissert/canvasview/DCanvasLink.cpp
--- kdissert-1.0.5-orig/src/kdissert/canvasview/DCanvasLink.cpp	2005-10-21 22:43:14.000000000 +0000
+++ kdissert-1.0.5/src/kdissert/canvasview/DCanvasLink.cpp	2005-12-21 21:16:39.000000000 +0000
@@ -13,6 +13,7 @@
  ***************************************************************************/
 
 #include <math.h>
+#include <stdlib.h>
 #include <qpainter.h>
 #include <qcolor.h>
 #include <kdebug.h>
